install_mac.py

```diff
@@ -103,11 +103,16 @@ def install_brew_packages(disable: List[str]):
             packages.remove(p)
 
     for p in packages:
-        if p not in brew_list.split('\n'):
-            brew('install', p, '--force')
+        if p in disable:
+            logger.info(f'skipping {p}')
+            continue
+
+        if p not in brew_list:
+            confirm_install(f'install {p}', brew['install', p, '--force'])
 
-    git('lfs', 'install')
-    sudo('git', 'lfs', 'install', '--system')
+    if 'git-lfs' not in brew_list and 'git-lfs' in brew('list').split('\n'):
+        git('lfs', 'install')
```

Comment:
missed a spot? or do we assume git lfs is a mandatory requirement?

Reply:
check if `git-lfs` just installed through brew